packages/firebase/index.d.ts

```diff
@@ -6949,15 +6949,49 @@ declare namespace firebase.messaging {
    *
    */
   interface Messaging {
+    /**
+     * Deletes the only registration token associated with this messaging instance and unsubscribes
```

Comment:
Remove the word `only`, it is unnecessary.

Reply:
+1